bin/license.py

```diff
@@ -0,0 +1,387 @@
+""",
+    "CC0-1.0 License": """The following components are provided under the CC0-1.0 License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "CDDL License": """The following components are provided under the CDDL License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Eclipse Public License": """The following components are provided under the Eclipse Public License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Public Domain License": """The following components are provided under the Public Domain License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Mozilla Public License Version 2.0": """The following components are provided under the Mozilla Public License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Unicode License": """The following components are provided under the Unicode License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Go License": """The following components are provided under the Go License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+"""
+}
+licenses_map = {
+    "Apache 2.0 License": [],
+    "MIT License": [],
+    "BSD License": [],
+    "CC0-1.0 License": [],
+    "CDDL License": [],
+    "Eclipse Public License": [],
+    "Public Domain License": [],
+    "Mozilla Public License Version 2.0": [],
+    "Unicode License": [],
+    "Go License": [],
+    "Other License": []
+}
+
+for _ in licenses:
+    if "org.apache.seatunnel" in _:
+        continue
+    if "Unknown license" in _:
+        for k, v in dependency_licenses_map.items():
+            if k in _:
+                _ = v
+                break
+    _ = _.strip(" ")
+    if _ == '\n':
+        continue
+    if '(' not in _ or ')' not in _:
+        continue
+    # (Apache 2.0 License) Spark Project Tags (org.apache.spark:spark-tags_2.11:2.4.0 - http://spark.apache.org/)
+    items = _.split(") ")
+    if len(items) != 2:
+        continue
+    type = items[0]
+    l = None
+    for k in licenses_keyword_map:
+        for keyword in licenses_keyword_map[k]:
+            if keyword in type:
+                l = k
+                break
+    if l is None:
+        l = "Other License"
+    licenses_map[l].append(_.strip('\n'))
+
+if len(licenses_map["Other License"]) != 0:
+    for other_license in licenses_map["Other License"]:
+        print(other_license)
+    print("There are some licenses are unknown!")
+    print("Please confirm the license by finding LICENSE file in the corresponding Jar file and maintain it in the unknown_licenses_map instance.")
```

Comment:
There is no `unknown_licenses_map` instance, please reword the message to help contributor about how to deal with unknown licenses

Reply:
Nice catch :+1: